Makefile

```diff
@@ -0,0 +1,19 @@
+REGISTRY_NAME=mselvara
+IMAGE_NAME=fcplugin
+IMAGE_VERSION=v1.0.0
```

Comment:
maybe this should be canary.  As a separate PR later, we can use csi-release-tools instead